entity-framework/core/modeling/inheritance.md

```diff
@@ -8,28 +8,49 @@ uid: core/modeling/inheritance
 ---
 # Inheritance
 
-Inheritance in the EF model is used to control how inheritance in the entity classes is represented in the database.
+EF can map a .NET type hierarchy to a database. This allows you to write your .NET entities in code as usual, using base and derived types, and have EF seamlessly create the appropriate database schema, issue queries, etc. The actual details of how a type hierarchy is mapped are provider-dependent; this page describes inheritance support in the context of a relational database.
 
-## Conventions
+At the moment, EF Core only supports the table-per-hierarchy (TPH) pattern. TPH uses a single table to store the data for all types in the hierarchy, and a discriminator column is used to identify which type each row represents.
 
-By default, it is up to the database provider to determine how inheritance will be represented in the database. See [Inheritance (Relational Database)](relational/inheritance.md) for how this is handled with a relational database provider.
+> [!NOTE]
+> The table-per-type (TPT) and table-per-concrete-type (TPC), which are supported by EF6, are not yet supported by EF Core. TPT is a major feature planned for EF Core 5.0.
 
-EF will only setup inheritance if two or more inherited types are explicitly included in the model. EF will not scan for base or derived types that were not otherwise included in the model. You can include types in the model by exposing a *DbSet\<TEntity>* for each type in the inheritance hierarchy.
+## Included hierarchy types
```

Comment:
Default hierarchy mapping?

Reply:
Well, this section does discuss how to use the fluent API to include types which aren't included "by default"...

How about `Hierarchy type mapping`? Am open to other ideas too.

Reply:
Or `Hierarchy entity type mapping`?

Reply:
Entity type hierarchy mapping?

Reply:
Sold.